mushr_hardware/mushr_hardware/launch/racecar-uw-nano/sensors.launch

```diff
@@ -1,11 +1,17 @@
 <!-- -*- mode: XML -*- -->
 <launch>
 
-  <arg name="racecar_version" />
+  <arg name="racecar_version" default="racecar-uw-nano"/>
+  <arg name="car_name"/>
```

Comment:
Does having a default make sense?

Reply:
Oops; removing the default for racecar_version, adding one for car_name.